source/mongodump.txt

```diff
@@ -624,20 +624,26 @@ Options
 
 .. option:: --oplog
 
-   Creates a file named :file:`oplog.bson` as part of the
-   ``mongodump`` output. The ``oplog.bson`` file, located in
-   the top level of the output directory, contains oplog entries that
-   occur during the ``mongodump`` operation. This file provides
-   an effective point-in-time snapshot of the state of a
-   ``mongod`` instance. To restore to a specific point-in-time
-   backup, use the output created with this option in conjunction with
-   :option:`mongorestore --oplogReplay`.
-   
+   Creates a file named :file:`oplog.bson` as part of the ``mongodump``
+   output. The ``oplog.bson`` file, located in the top level of the
+   output directory, contains oplog entries that occur during the
+   ``mongodump`` operation.
+   
+   To apply oplog entries from the ``oplog.bson`` file in the restore
+   operation, use :option:`mongorestore --oplogReplay`. You can use
+   ``mongodump --oplog`` together with ``mongorestore --oplogReplay`` to
+   ensure the data is current and has all the writes that occurred
+   during the dump. However, if any writes occur after ``mongodump``
+   completes, those writes aren't captured in that dump operation.
+
    Without ``--oplog``, if there are write operations during the dump
    operation, the dump will not reflect a single moment in time. Changes
    made to the database during the update process can affect the output of
    the backup.
-   
+
+   To back up individual replica sets, use ``--oplog``. To back up
```

Comment:
Can we add "while still accepting writes" after "individual replica sets"?

Reply:
Done